addon/components/bread-crumb.js

```diff
@@ -14,6 +14,7 @@ export default Component.extend({
   layout,
   tagName: 'li',
   classNameBindings: ['crumbClass'],
+  i18n: false,
```

Comment:
Or, a Mixin/base route that is responsible for setting `breadCrumb` in a lifecycle method.

```js
export default Ember.Mixin.create({
  i18n: Ember.inject.service(),

  init() {
    this._super(...arguments);

    if (!this.breadCrumb) {
      this.breadCrumb = {};
    }
  },

  afterModel() {
    const key = get(this, 'i18nTitle');

    if (key) {
      const breadCrumb = get(this, 'breadCrumb');

      Object.assign(breadCrumb, {
        title: get(this, 'i18n').t(key)
      });
    }

    return this._super(...arguments);
  }
});
```